ilastik/config.py

```diff
@@ -83,7 +83,7 @@
 
 @dataclass
 class RuntimeCfg:
-    tiktorch_executable: Optional[str] = None
+    tiktorch_executable: Optional[List[str]] = None
```

Comment:
Is it possible to have multiple tiktorch executables? If this is really possible, then it probably makes sense to drop `Optional` and default to `field(default_factory=list)`.

Reply:
The singular here is correct - it is a single exectutable. But it could be something like `['python', '-m', 'tiktorch']`, or anything alike. But sure, can definitely use the default factory - as far as I can see the code does not rely on it being `None` if not provided, something "falsy" is good enough.